### PR TITLE
Make opensource_clubs about page images editable.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: coss/opensource_clubs/models.py [hunk 1]
```diff
@@ -200,9 +200,27 @@ class AboutPage(Page):
         on_delete=models.SET_NULL,
         related_name='+'
     )
+    og_image = models.ForeignKey(
```
Comment (akatsoulas) [inline coss/opensource_clubs/models.py]: s/og_image/opengraph_image

**Comment by akatsoulas** [comment]
According to the initial sketch, if there is not an image, the default should be the `alex-wong`. Let's add that as a default in the model.

**Comment by akatsoulas** [comment]
r+wc

**Pull request merged**